docs/resources/service.md

```diff
@@ -73,12 +78,17 @@ resource "docker_network" "test_network" {
   driver = "overlay"
 }
 
+resource "docker_image" "foo" {
```

Comment:
if the image already exists somewhere, would this be better represented as a `data` source?  Or is the idea that having it be a `resource` will check whether the image:tag has a new hash associated with it?

Reply:
This could also be an option. ATM there is not data-source for an image. Yes, the `pull_triggers` will check for a newer version of the image

Reply:
I added a `docker_image` datasource in a3a236145b62bee0f28ea1a904ae988cd3285dcb